sdk/monitor/azure-monitor-opentelemetry-exporter/src/main/java/com/azure/monitor/opentelemetry/exporter/ExportOptions.java

```diff
@@ -0,0 +1,118 @@
+package com.azure.monitor.opentelemetry.exporter;
+
+import com.azure.core.annotation.Fluent;
+import com.azure.core.credential.TokenCredential;
+import com.azure.core.http.HttpClient;
+import com.azure.core.http.HttpPipeline;
+import com.azure.core.http.policy.HttpLogDetailLevel;
+import com.azure.core.http.policy.HttpLogOptions;
+import com.azure.core.http.policy.HttpPipelinePolicy;
+import com.azure.core.util.ClientOptions;
+import com.azure.monitor.opentelemetry.exporter.implementation.configuration.ConnectionString;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+/**
+ * The options to configure the Azure Monitor export.
+ */
+@Fluent
+public final class ExportOptions {
+
+    ConnectionString connectionString;
```

Comment:
This field and the following ones are package-private . So these fields are accessible by `AzureMonitorExporterBuilder` but not by the user.